### PR TITLE
Update failure message for too-few blocks case

### PR DESCRIPTION
**Pull request by balderdash**

### CODE REVIEW

**Comment by Hamms** [comment]
would you also double-check to see if we're using `msg.tooFewBlocksMsg` anywhere else in the codebase and if not, remove it from the i18n file?

File: apps/src/feedback.js [hunk 1]
```diff
@@ -501,7 +501,9 @@ FeedbackUtils.prototype.getFeedbackMessage_ = function(options) {
         }
         break;
       case TestResults.TOO_FEW_BLOCKS_FAIL:
-        message = options.level.tooFewBlocksMsg || msg.tooFewBlocksMsg();
+        message = options.level.tooFewBlocksMsg ||
+            options.level.levelIncompleteError ||
+            msg.levelIncompleteError();
```
Comment (Hamms) [inline apps/src/feedback.js:508]: I'm not a big fan of the repeated `options || msg` lines here and in the `LEVEL_INCOMPLETE_FAIL` case, but at the same time I'm not sure if there's a better way of doing this (that doesn't involve something gross like a conditional `break`). Maybe just throw in a comment making it absolutely explicit that we're defaulting to the `LEVEL_INCOMPLETE_FAIL` case?

**Comment by Hamms** [comment]
One nitpicky comment that you're free to ignore; otherwise LGTM!

File: apps/src/feedback.js (hunk 1 above)
Comment (balderdash) [inline apps/src/feedback.js:508]: Yeah, the other possibilities I could think of seemed needlessly complicated and/or ugly. Added a comment.

**Pull request merged**